src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +24,8 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        LOGGER.info("data was successfully fetched from DB");
+        LOGGER.info("products for user id {} were successfully fetched from DB: {}, {}, {}.",
+                userId, iphone.getTitle(), macBook.getTitle(), xiaomi.getTitle());
```

Comment:
ok, but in real life we get data from db and don't have such variables like `iphone` or `macBook`
so let's log `products` variable (we can override `toString` for `Product`)